mmv1/third_party/terraform/resources/resource_compute_instance_template.go.erb

```diff
@@ -1176,6 +1188,7 @@ func resourceComputeInstanceTemplateCreate(d *schema.ResourceData, meta interfac
 	if err != nil {
 		return err
 	}
+	resourcePolicies := convertAndMapStringArr(d.Get("resource_policies").([]interface{}), GetResourceNameFromSelfLink)
```

Comment:
We can have a expand function for `resourcePolicies`

Reply:
Done

Reply:
Sorry, it was my fault, I haven't properly added file to commit. Fixed now.